lib/UnrarXLib/win32acl.cpp

```diff
@@ -7,9 +7,8 @@ static bool ReadSacl=false;
 #ifndef SFX_MODULE
 void ExtractACL(Archive &Arc,char *FileName,wchar *FileNameW)
 {
-#if defined(_XBOX) || defined (_LINUX)
   return;
```

Comment:
function returns now on every platform without any code executed
If this is relay wanted remove code below.